documentation/general/dash-sonic-hld.md

```diff
@@ -1017,4 +1084,17 @@ For the example configuration above, the following is a brief explanation of loo
 			Overlay DIP 2603:10e1:100:2::3402:206 (No transformation, provided as part of mapping)
 		f. Second Action is Static NVGRE encap with GRE key '100'. 
 		g. Underlay DIP shall be 50.2.2.6 (from mapping), Underlay SIP shall be 55.1.2.3 (from ENI)
-	
+
+	3. Packet destined to 10.2.0.9 from 10.1.1.2:
+		a. LPM lookup hits for entry 10.2.0.6/24
+		b. The action in this case is "vnet"
+		c. Next lookup is in the mapping table and mapping table action here is "privatelinknsg"
+		d. First Action for "privatelink" is 4to6 transposition
+		e. Packet gets transformed as: 
+		 	For Overlay SIP, using ENI's "pl_sip_encoding": "0x0020000000000a0b0c0d0a0b/0x002000000000ffffffffffff" -> Overlay SIP fd30:108:0:0a0b:0c0d:0a0b:a01:102;	
+			Overlay DIP 2603:10e1:100:2::3402:206 (No transformation, provided as part of mapping)
+		f. Second Action is Static NVGRE encap with GRE key '100'. 
+		g. Underlay DIP shall be 50.2.2.6 (from mapping), Underlay SIP shall be 55.1.2.3 (from ENI)
+		h. Third Action is Appliance Encap for id 22
+		i. Packet shall be encapsulated with Outer DIP as 100.8.1.2 and SIP as VIP of this originating appliance card with VNI of 101. 
+		j. Inbound flow shall be similar to PL and outer encap shall be of the SLB MUX and not of the NSG appliance.
```

Comment:
We have not described inbound flow for a PL in any of the examples.  Should we not describe it?  Also, what happens when the flow fixup happens due to the ICMP redirect?  What will be the resulting inbound flow look like when the packet comes through the fast-path?